packages/next/src/client/dev/dev-build-watcher.ts

```diff
@@ -6,18 +6,19 @@ export default function initializeBuildWatcher(
   toggleCallback: any,
   position = 'bottom-right'
 ) {
-  const shadowHost = document.createElement('div')
+  const type = 'div' as string
```

Comment:
Best to use type annotations rather than type-casting in such cases. Consider the following two cases, and note they they are _not_ equivalent:

```ts
const foo: "foo" = "bar";
const foo = "bar" as "foo";
```

Both examples result in `foo` being `"foo"`-literal typed, but only the first example correctly throws a type-error. See Typescript REPL example here: https://www.typescriptlang.org/play?#code/N4KABBYMYPYHYGcAuYBmMYC4wCJ0xzAF5cAjAQwCccBuEAXxFEmnmTQ2LKsPIV3y0GQA

Reply:
I think this was accidentally left in after trying a few things.

It can be reverted. See PR https://github.com/vercel/next.js/pull/54221